#! /bin/sh /usr/share/dpatch/dpatch-run
## 96_add_mootols_more.dpatch by Alexander Wirt <formorer@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: use mootools-more

@DPATCH@
diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch' '--exclude=.hg' '--exclude=_darcs' '--exclude=.bzr' icinga-1.2~/html/menu.html.in icinga-1.2/html/menu.html.in
--- icinga-1.2~/html/menu.html.in	2010-10-05 10:56:45.000000000 +0200
+++ icinga-1.2/html/menu.html.in	2010-10-05 22:51:57.071771876 +0200
@@ -8,6 +8,7 @@
 	<link rel="stylesheet" type="text/css" href="stylesheets/interface/menu.css" media="screen, projection" />
 	<link rel="shortcut icon" type="image/x-icon" href="images/favicon.ico" />
 	<script type="text/javascript" src="js/mootools.js"></script>
+	<script type="text/javascript" src="js/mootools-more.js"></script>
 	<script type="text/javascript" src="js/menu.js"></script>
 </head>
 <body>
